platform/darwin/scripts/generate-style-code.js

```diff
@@ -281,6 +281,18 @@ global.propertyDoc = function (propertyName, property, layerType, kind) {
             doc += `\n\nThis attribute corresponds to the <a href="https://www.mapbox.com/mapbox-gl-style-spec/#${anchor}"><code>${property.original}</code></a> layout property in the Mapbox Style Specification.`;
         }
     }
+    if (property["property-function"]) {
+        doc += '`\n\nThis property can be set to an `MGLStyleConstantValue`, an `MGLCameraStyleFunction` using an `MGLInterpolationModeExponential` or `MGLInterpolationModeInterval` ' +
+               'interpolation mode, an `MGLSourceStyleFunction` using an `MGLInterpolationModeExponential`, `MGLInterpolationModeInterval`, `MGLInterpolationModeCategorical, or `MGLInterpolationModeIdentity` ' + 
+               'interpolation mode, or an `MGLCompositeStyleFunction` using an `MGLInterpolationModeExponential`, `MGLInterpolationModeInterval` or `MGLInterpolationModeCategorical` interpolation mode`.';
```

Comment:
Which leads to this, on `MGLCircleStyleLayer.circleOpacity`:

![screen shot 2017-02-08 at 2 08 10 pm](https://cloud.githubusercontent.com/assets/1198851/22752904/2a3e40f0-ee08-11e6-820b-8e6c67301c1c.png)